homeassistant/components/http/__init__.py

```diff
@@ -289,6 +295,19 @@ def register_static_path(self, url_path, path, cache_headers=True):
                               self.ssl_certificate, error)
                 context = None
                 return
+
+            try:
+                if self.ssl_peer_certificate:
+                    context.verify_mode = ssl.CERT_REQUIRED
+                    context.load_verify_locations(
+                        cafile=self.ssl_peer_certificate)
+            except OSError as error:
+                _LOGGER.error(
+                    "Could not read peer SSL certificate from %s: %s",
+                    self.ssl_peer_certificate, error)
+                context = None
```

Comment:
Why set it to `None` ?

Reply:
same as above, TBH I copied this snippet blindly from how we do for server-side certs above. since it does not makes sense here (and I agree it doesn't), should I remove it from line 296 too?